app/controllers/DatasetController.scala

```diff
@@ -418,7 +418,7 @@ class DatasetController @Inject()(userService: UserService,
           datastoreClient <- datasetService.clientFor(dataset)(GlobalAccessContext)
           targetMagSelectedBbox: BoundingBox = request.body.surroundingBoundingBox / request.body.mag
           _ <- bool2Fox(targetMagSelectedBbox.size.sorted.z <= 1024 && targetMagSelectedBbox.size.sorted.y <= 1024) ?~> s"Target-mag selected bbox must be smaller than 1024×1024×depth (or transposed), got ${targetMagSelectedBbox.size}"
-          _ <- bool2Fox(targetMagSelectedBbox.size.sorted.x <= 12) ?~> s"Target-mag selected bbox depth must be at most 12"
+          _ <- bool2Fox(targetMagSelectedBbox.size.sorted.x <= 16) ?~> s"Target-mag selected bbox depth must be at most 16"
```

Comment:
not sure how we could unify this with the constant in the frontend?

Reply:
let’s add comments cross-linking the two spots. I think extracting this to a common source is overkill. Same as why I closed https://github.com/scalableminds/webknossos/issues/5223